executor/adapter.go

```diff
@@ -379,54 +379,37 @@ func (a *ExecStmt) LogSlowQuery(txnTS uint64, succ bool) {
 	sessVars := a.Ctx.GetSessionVars()
 	sql = QueryReplacer.Replace(sql) + sessVars.GetExecuteArgumentsInfo()
 
-	connID := sessVars.ConnectionID
-	currentDB := sessVars.CurrentDB
-	var tableIDs, indexIDs string
-	if len(sessVars.StmtCtx.TableIDs) > 0 {
-		tableIDs = strings.Replace(fmt.Sprintf("table_ids:%v ", a.Ctx.GetSessionVars().StmtCtx.TableIDs), " ", ",", -1)
+	var tableNames, indexNames string
+	if len(sessVars.StmtCtx.TableNames) > 0 {
+		tableNames = strings.Join(sessVars.StmtCtx.TableNames, ",")
```

Comment:
Table names and index names might be too long to log. It can quickly enlarges the file size of tidb log or tidb slow query log.

Reply:
How about providing a sql command to query table name from table id and index name from table id and index id? So that users don't need to query the HTTP API and search the result from a huge JSON response.

Reply:
now, we can gett them via this two sqls:

`select * from TABLES where table_name = ?`

`select * from TIDB_INDEXES where table_name = ?`

Reply:
So, prefer use table id and index id?  
We wont print `tables` in the slow log, only print `indexs`.
But Index ids may be same in different table.

Reply:
We print the tableIDs and indexIDs before, so I think to keep it unchanged is ok.

Reply:
@zz-jason We discuss before is use index/table name instead of id. In discuss document also use index name. Are you sure to use index id back?

Reply:
Maybe we need a discussion.

Reply:
done. @zz-jason .